src/class/number.rs

```diff
@@ -742,17 +750,17 @@ where
     }
 }
 
```

Comment:
Doesn't the selection of the PyNumberProtocolImpl impl depend upon specialisation? This branch doesn't compile for me on nightly 2019-11-09.